lti_redirect/utils/debugpy.py

```diff
@@ -0,0 +1,18 @@
+from decouple import config
+from distutils.util import strtobool
+import debugpy
+
+
+def forcebool(bool_str):
+    if isinstance(bool_str, str):
+        return strtobool(bool_str)
```

Comment:
That is, all together…

```suggestion
import debugpy


def forcebool(bool_str):
    if isinstance(bool_str, str):
        return str(value).lower() in ('true', '1', 'yes', 'on')
```

Reply:
@lsloan I cannot take this commit since the the function parameter is `bool_str` but the suggested code is written with `value`. 

But I have made the code commit as suggested by @jonespm

Reply:
Yeah, I should've changed the argument.